app/src/main/java/com/example/vladzakharo/project/ActivityFragments.java

```diff
@@ -0,0 +1,89 @@
+package com.example.vladzakharo.project;
+
+
+import android.content.Intent;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.MenuItem;
+
+public class ActivityFragments extends AppCompatActivity {
+
+    FragmentManager fm;
+
+    private String mTitle;
+    private String mMessage;
+    private String mFruit;
+    private boolean isCheck;
+
+    private static final String TITLE = "title";
+    private static final String MESSAGE = "message";
+    private static final String SPINNER = "spinner";
+    private static final String CHECK = "check";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_fragments);
+
+        mTitle = getIntent().getStringExtra("title");
+        mMessage = getIntent().getStringExtra("message");
+        mFruit = getIntent().getStringExtra("fruit");
+        isCheck = getIntent().getExtras().getBoolean("check");
+
+        getSupportActionBar().setTitle(mTitle);
+
+        if (isCheck){
+            fm = getSupportFragmentManager();
+            Fragment fragmentOne = fm.findFragmentById(R.id.fragmentF1);
+            if(fragmentOne == null){
+                fm.beginTransaction()
+                        .add(R.id.fragmentF1, FragmentTwo.newInstance(mMessage))
+                        .commit();
+            }
+
+            Fragment fragmentTwo = fm.findFragmentById(R.id.fragmentF2);
+            if(fragmentTwo == null){
+                fm.beginTransaction()
+                        .add(R.id.fragmentF2, FragmentOne.newInstance(mFruit))
+                        .commit();
+            }
+        }else{
+            fm = getSupportFragmentManager();
+            Fragment fragmentOne = fm.findFragmentById(R.id.fragmentF1);
+            if(fragmentOne == null){
+                fm.beginTransaction()
+                        .add(R.id.fragmentF1, FragmentOne.newInstance(mMessage))
```

Comment:
Can be simplified. For example:`isCheck ? FragmentTwo.newInstance(mMessage) : FragmentOne.newInstance(mMessage)`.